api/.env

```diff
@@ -52,3 +52,10 @@ MAIL_FROM_NAME=spira API
 # RACKSPACE_KEY=null
 # RACKSPACE_CONTAINER=null
 # RACKSPACE_REGION=null
+
+PROVIDER_FACEBOOK_CLIENT_ID={$PROVIDER_FACEBOOK_CLIENT_ID}
+PROVIDER_FACEBOOK_CLIENT_SECRET={$PROVIDER_FACEBOOK_CLIENT_SECRET}
+PROVIDER_TWITTER_CLIENT_ID={$PROVIDER_TWITTER_CLIENT_ID}
+PROVIDER_TWITTER_CLIENT_SECRET={$PROVIDER_TWITTER_CLIENT_SECRET}
+PROVIDER_GOOGLE_CLIENT_ID={$PROVIDER_GOOGLE_CLIENT_ID}
+PROVIDER_GOOGLE_CLIENT_SECRET={$PROVIDER_GOOGLE_CLIENT_SECRET}
```

Comment:
Can you create the `.secret.env.example` file for these values and link the (.gitignored) non example file to the php & devtools containers?